test/testdata/deployednettemplates/recipes/fastnet/genesis.json

```diff
@@ -0,0 +1,104 @@
+{
+	"NetworkName": "",
```

Comment:
Might as well name this alphanet right from the start!

Reply:
Renamed to alphanet.